test/schemas/tenantSchema.ts

```diff
@@ -21,7 +21,7 @@ export default {
         User: {
             pk:         { type: String, value: 'Account#${accountId}' },
             sk:         { type: String, value: '${_type}#${id}' },
-            accountId:  { type: String, required: true },
+            accountId:  { type: String },
             id:         { type: String, generate: 'ulid' },
```

Comment:
Why remove required here?  As it is part of the PK, it should be required.

Reply:
This issue is now visible because the model is now inferred so it means that in contexts.ts we cant create a user without the account id so I just made it not required. The alternative is the fix the create function here https://github.com/sensedeep/dynamodb-onetable/blob/main/test/context.ts#L76

Reply:
Okay, that makes sense if the accountId is coming from the context.